TileStache/Goodies/VecTiles/transform.py

```diff
@@ -389,6 +389,14 @@ def place_ne_capital(shape, properties, fid, zoom):
     return shape, properties, fid
 
 
+def place_population_int(shape, properties, fid, zoom):
+    population_str = properties.get('population')
```

Comment:
Do we want to `pop` the population property so that if the parsing fails we don't leave a `str` property?